data_driven_acquisition/static/css/data_driven_acquisition.css

```diff
@@ -86,6 +54,7 @@
   background: none;
   font-weight: normal;
   display: block;
+  color: #565c65;
```

Comment:
Added in #19, standardizing the text color of sidebar menu items between folders and files.